client/src/components/ui/itemrow/InboxRow.svelte

```diff
@@ -7,23 +7,37 @@
     import { IconSize, ContextPayload, RowType, Events } from '../../types.ts';
     import { Document } from '../../../../../model/src/document.ts';
     import { Category } from '~model/category.ts';
+    import { Snapshot } from '~model/snapshot.ts';
 
     export let iconSize: IconSize;
     export let doc: Document['id'];
     export let category: Category['name'];
     export let title: Document['title'];
+    export let creation: Snapshot['creation'];
 
     const dispatch = createEventDispatcher();
     const rowEvent: ContextPayload = {
         ty: RowType.Inbox,
         id: doc,
     };
+
+    function redirectHandler() {
+        window.location.href = `/track?id=${doc}`;
+    }
+    
+    const docDisplay = `${doc.slice(0,5)}...${doc.slice(-5)}`;
```

Comment:
This hand-rolled logic is already built into CSS with the [`text-overflow`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow) property.